docs/content/configuration/csi_ug_config_create_vol_snapshotclass_topology.md

```diff
@@ -0,0 +1,49 @@
+# Creating a VolumeSnapshotClass with topology awareness
+
+When using the CSI Topology feature, different parameters must be taken into account when creating a VolumeSnapshotClass YAML file with specific `by_system_id` requirements. Use this information to help define a VolumeSnapshotClass that is topology aware and enables the creation and deletion of volume snapshots.
+
+**Note:** 
+  - For information and parameter definitions that are not related to topology awareness, be sure to see the information provided in [Creating a VolumeSnapshotClass](csi_ug_config_create_vol_snapshotclass.md), in addition to the current section.
+  
+  - IBM® FlashCopy® function is referred to as the more generic volume snapshots and cloning within this documentation set. Not all supported products use the FlashCopy function terminology.
+
+In order to enable creation and deletion of volume snapshots for your storage system, create a VolumeSnapshotClass YAML file, similar to the following `demo-volumesnapshotclass-config-secret.yaml`.
+
+When configuring the file, be sure to use the same array secret and array secret namespace as defined in [Creating a Secret with topology awareness](csi_ug_config_create_secret_topology.md).
+
+**Note:**
+  - The `snapshot_name_prefix` parameter is optional.
+  - The `CSI` prefix is added as default if not specified by the user.
+
+  The `by_system_id` parameter is optional and values, such as the `pool`, `SpaceEfficiency`, and `volume_name_prefix` may all be specified in the following places, and within the following hierarchical order:
+1. Within the parameter, per system.
+2. Outside of the parameter, as a cross-system default.
+3. Both inside and outside of the parameter, as specified above.
+  
+    When the parameter is defined in both places, it is determined by the default parameter value.
```

Comment:
@ArbelNathan is this true? what exactly does it mean?

Reply:
updated per our other conversation - please approve.

Reply:
updated again per my conversation with @ArbelNathan 
DONE